tequilapi/endpoints/proposals.go

```diff
@@ -226,6 +237,7 @@ func (pe *proposalsEndpoint) List(resp http.ResponseWriter, req *http.Request, p
 		proposalsRes.Proposals = append(proposalsRes.Proposals, proposalToRes(p))
 	}
 
+	fetchConnectCounts := req.URL.Query().Get("fetch_connect_counts")
```

Comment:
Probably now `fetch_metrics` is better name for that it does. Or do do granular fetch approach with query:
`&fetch_metric[connect_counts]=false&fetch_metric[monitoring_failed]=true`

Reply:
Renamed.